AppCenter/AppCenterTests/MSModelTestsUtililty.h

```diff
@@ -6,24 +6,163 @@
 
 /**
  * Get dummy values for device model.
+ *
  * @return Dummy values for device model.
  */
 + (NSDictionary *)deviceDummies;
 
 /**
- * Get a dummy device model.
- * @return A dummy device model.
+ * Get dummy values for common schema extensions.
+ *
+ * @return Dummy values for common schema extensions.
  */
-+ (MSDevice *)dummyDevice;
++ (NSMutableDictionary *)extensionDummies;
+
+/**
+ * Get dummy values for common schema user extensions.
+ *
+ * @return Dummy values for common schema user extensions.
+ */
++ (NSDictionary *)userExtensionDummies;
+
+/**
+ * Get dummy values for common schema location extensions.
+ *
+ * @return Dummy values for common schema location extensions.
+ */
++ (NSDictionary *)locExtensionDummies;
+
+/**
+ * Get dummy values for common schema os extensions.
+ *
+ * @return Dummy values for common schema os extensions.
+ */
++ (NSDictionary *)osExtensionDummies;
+
+/**
+ * Get dummy values for common schema app extensions.
+ *
+ * @return Dummy values for common schema app extensions.
+ */
++ (NSDictionary *)appExtensionDummies;
+
+/**
+ * Get dummy values for common schema protocol extensions.
+ *
+ * @return Dummy values for common schema protocol extensions.
+ */
++ (NSDictionary *)protocolExtensionDummies;
+
+/**
+ * Get dummy values for common schema net extensions.
```

Comment:
I would specify ".NET" (with caps) in comment.  Could be renamed to `dotNetExtensionDummies` too.  Or am I misunderstanding what this is supposed to be.

Reply:
@jwallra It is network not .NET. See [MSNetExtension.h](https://github.com/Microsoft/AppCenter-SDK-Apple/blob/master/AppCenter/AppCenter/Internals/Model/CommonSchema/MSNetExtension.h).

Reply:
Still would prefer for it to be written out in that case